package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "fix": "npm run lint:es -- --fix && npm run lint:css -- --fix",
-    "htmlhint": "htmlhint \"**/*.html\" \"**/*.template\" -i \"common/**/*.html\" -f unix",
+    "htmlhint": "htmlhint \"aria-practices.html\" -i \"**/*.html\" \"**/*.template\" -i \"common/**/*.html\" -f unix",
```

Comment:
`-i "**/*.html"` means files matching `**/*.html` should be ignored. Is that the intended change here?

Reply:
https://htmlhint.com/docs/user-guide/usage/options#ignore

Reply:
I think it would be good to replace `-i` with `--ignore` here for clarity. (And maybe `--format` instead of `-f`.)

Reply:
The larger glob should already be capturing `aria-practices.html`

Reply:
Oops, yeah I didn't mean "ignore"

Reply:
```suggestion
    "htmlhint": "htmlhint \"**/*.html\" \"**/*.template\" --ignore \"common/**/*.html\" --format unix",
```